packages/liveblocks-redux/package.json

```diff
@@ -25,7 +25,7 @@
     "collaborative"
   ],
   "scripts": {
-    "build": "rollup -c",
+    "build": "rollup -c && cp ./package.json ./README.md ./lib",
```

Comment:
Ah yes!

For what it's worth, I'm doing something similar in the build script for decoders. For example, I'm also [copying some files manually](https://github.com/nvie/decoders/blob/main/bin/build.sh#L72), changing [all references to `dist/`](https://github.com/nvie/decoders/blob/main/bin/build.sh#L89) from the original `package.json`, and [removing keys from `package.json` that don't carry value on NPM](https://github.com/nvie/decoders/blob/main/bin/build.sh#L82-L87). The result is a beautiful package though, for someone who ever inspects it. I'm not using rollup, as I could never figure out how to make it work with all the environments at once (TypeScript, Flow, ES modules, CJS, etc.), so I'm basically creating 4 versions of each original source file (`.ts`, `.js.flow`, `.mjs`, and `.js`, respectively). For the `.mjs` files, that means having to re-write the imports inside those files, [using this hack](https://github.com/nvie/decoders/blob/main/bin/build.sh#L57) which rewrites `import ... from 'foo'` to `import ... from 'foo.mjs'`.

It's basically very manually crafting the contents of what you'd see on the "Explore" tab on https://www.npmjs.com/package/decoders. It's a shame that this is so hard to fully control, and unclear what best practices are. It took me a lot of beta releases of 2.0 to get the final structure right, but the end result is beautiful, works in all environments, and is fully tree-shakable.

Part of the trick was realizing that `package.json` does too many things. It's used for local development, but also acts as the "index" for what gets published on NPM.

Eventually I'd be in favor of wrapping this up into a light generic build script like with decoders, where we can do more tweaks/cleanups pre-publishing.